<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1340</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="mainWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1260</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <widget class="QPushButton" name="closebtn">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="minibtn">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="leftWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>536</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#leftWidget{border-image: url(:/left);}</string>
    </property>
    <widget class="QLabel" name="Hour_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>72</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>H</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Hour_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>72</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>H</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Min_1">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>72</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>M</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Min_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>120</y>
       <width>72</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>M</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="num">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>260</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>一</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="week">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>01</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="month">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>250</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>四</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="datelabel">
     <property name="geometry">
      <rect>
       <x>241</x>
       <y>250</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>10</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="clock">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>130</y>
       <width>93</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="manage">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>350</y>
       <width>181</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="guide">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>350</y>
       <width>181</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="arrange">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>530</y>
       <width>181</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="log">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>530</y>
       <width>181</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="acceleration">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>61</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="mini">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="rightWidget" native="true">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>731</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#rightWidget{border-image: url(:/right);}</string>
    </property>
    <widget class="QPushButton" name="admin">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>660</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="expand">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>leftWidget</zorder>
   <zorder>rightWidget</zorder>
   <zorder>closebtn</zorder>
   <zorder>minibtn</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
